AlphaWalletWeb3.podspec

```diff
@@ -0,0 +1,35 @@
+#
+# Be sure to run `pod lib lint AlphaWalletWeb3.podspec' to ensure this is a
+# valid spec before submitting.
+#
+# Any lines starting with a # are optional, but their use is encouraged
+# To learn more about a Podspec see https://guides.cocoapods.org/syntax/podspec.html
+#
+
+Pod::Spec.new do |s|
+s.name             = "AlphaWalletWeb3"
+s.version          = "0.0.1"
+s.summary          = "Web3 implementation in vanilla Swift for iOS ans macOS"
+
+s.description      = <<-DESC
+Web3 implementation in vanilla Swift, intended for mobile developers of wallets, Dapps and Web3.0
+DESC
+
+s.homepage     = "https://github.com/AlphaWallet/AlphaWalletSDK"
+s.license      = { :type => "MIT", :file => "LICENSE" }
+s.author       = { "Vladyslav Shepitko" => "vladyslav.shepitko@gmail.com" }
+s.source       = { :git => 'git@github.com:AlphaWallet/alpha-wallet-ios.git', :tag => "#{s.version}" }
+
+s.swift_version = '4.2'
+s.module_name = 'AlphaWalletWeb3'
+s.ios.deployment_target = "10.0"
+s.osx.deployment_target = "10.11"
+s.source_files     = 'modules/AlphaWalletWeb3/AlphaWalletWeb3/**/*.{h,m,swift}'
+s.public_header_files = "AlphaWalletWeb3/**/*.{h}"
+s.pod_target_xcconfig = { 'DEFINES_MODULE' => 'YES' }
+
+s.dependency 'BigInt'#,'~> 3.1'
+s.dependency 'CryptoSwift'#, '~> 1.4'
+s.dependency 'secp256k1_ios', '~> 0.1.3'
+s.dependency 'PromiseKit', '~> 6.3.0'
```

Comment:
I see you literally did "make alpha wallet version of web3 pod" rather than to wrap the `web3swift` pod. Is it better?

I noticed this line (and a few others) has changed: https://github.com/AlphaWallet/web3swift/blob/6d7c01af26bcb75d8a02b6709b089e02ed99af98/web3swift/Promises/Classes/Promise+HttpProvider.swift#L93

Did you vendor the pod verbatim or did you make some changes? (reason being the pod is a bit more well-tested by now)

Reply:
Otherwise I don't have a strong preference whether we wrap or vendor and change web3swift. I probably like the latter a bit more anyway

Reply:
actually its kind of vendor the pod verbatim, removed deprecation from pod, and unused params. Removed some cases for error. refactored Promise+HttpProvider a little bit (no functional changes)